src/interface/src/app/home/welcome/welcome.component.scss

```diff
@@ -26,8 +37,14 @@ p {
 .flex {
   display: flex;
   align-items: flex-start;
+  gap:20px;
```

Comment:
This lack of space made me curious if we have a convention for linting CSS/SCSS -- I'm ok with having one or not having one, either way.

Reply:
Good point, we should! Lemme see if we can use prettier for this as well